aidies-project/client/src/components/CampaignCreate.js

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useLocation } from 'react-router-dom';
+import { useParams, useHistory } from 'react-router-dom';
+
+import { getOneCampaign, bid, setLocation } from '../store/campaign';
+import { DetermineBid, DetermineTimeRemaining } from './CampaignUtils';
+import { makeStyles } from '@material-ui/core/styles';
+import { Card, Typography, Button} from '@material-ui/core';
+import '../css/campaign.css';
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        '& > *': {
```

Comment:
Interesting! Looking at this helped me learned something new too - seeing this in your PR made me read this [Stack Overflow post](https://stackoverflow.com/questions/63805961/react-jss-how-to-style-custom-html-tag)!

Reply:
Me too. Material UI is awesome, but customization is not very intuitive. I can't change the 'primary' color scheme for the life of me!!